frontend/src/components/utils/date.ts

```diff
@@ -0,0 +1,18 @@
+const months = [
+  'Jan',
+  'Feb',
+  'March',
+  'April',
```

Comment:
Generally these are shortened to Mar and Apr if we are taking abbreviations of the other months as well